### PR TITLE
[adding more logs]

### PR DESCRIPTION
**Pull request by b2mdevelopment**

### CODE REVIEW

**Comment by b2mdevelopment** [comment]
:+1: @ivanbokii

File: reyna/src/com/b2msolutions/reyna/Repository.java [hunk 1]
```diff
@@ -92,13 +97,18 @@ public void shrinkDb(long limit) {
...
             long dbSize = this.getDbSize(db);
 
+            Logger.v(TAG, "shrinkDb, dbSize: " + dbSize);
+            Logger.v(TAG, "shrinkDb, limit: " + limit);
             if (dbSize <= limit) {
+                Logger.v(TAG, "shrinkDb, dbSize <= limit, no SHRINK");
                 return;
             }
 
+            Logger.v(TAG, "shrinkDb, dbSize <= limit, SHRINK needed");
```
Comment (aflugge) [inline reyna/src/com/b2msolutions/reyna/Repository.java]: Should be "shrinkDb, dbSize > limit, SHRINK needed". Also as we can shrink multiple time within the loop, should we put this log message inside the loop as well?

File: reyna/src/com/b2msolutions/reyna/services/StoreService.java [hunk 2]
```diff
@@ -38,8 +38,10 @@ public static void setCellularDataBlackout(Context context, TimeRange range) {
     }
 
     public static void setStorageSizeLimit(Context context, long limit) {
+        Logger.v(TAG, "setStorageSizeLimit, limit: " + limit);
         limit = limit < MINIMUM_STORAGE_LIMIT ? MINIMUM_STORAGE_LIMIT : limit;
 
+        Logger.v(TAG, "setStorageSizeLimit,  limit: " + limit);
```
Comment (aflugge) [inline reyna/src/com/b2msolutions/reyna/services/StoreService.java]: Should we put in the log mesage that the first setStorageSizeLimit is before comparing to min value and the second one after?

**Comment by aflugge** [comment]
Other than comments :+1:

File: reyna/src/com/b2msolutions/reyna/services/StoreService.java (hunk 2 above)
Comment (b2mdevelopment) [inline reyna/src/com/b2msolutions/reyna/services/StoreService.java]: We have this one already
@Youhana-Hana

**Pull request merged**